resources/axon.rb

```diff
@@ -88,6 +88,15 @@
   when 'debian', 'ubuntu'
     ext = '.deb'
     eg_service_name = 'tw-eg-service'
+    if node["platform"].include?('debian')
```

Comment:
For Axon, we'll want to always install this when it's Debian or Ubuntu, regardless of version of the OS (for current agents at least).

Reply:
As discussed I will fix for Ubuntu later. Currently we have a release and need this fix, as of now our automation jobs are failing for Debian.

Reply:
Fixed for Ubuntu as well. Please review @kraigstrong